### PR TITLE
Add change log for release v1.2.0-rc1

### PR DESCRIPTION
**Pull request by lunny**
As title, prepare for release v1.2.0-rc1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,469 @@
...
+  * Fix broken table layout (#1344)
+  * LFS: Fix SSH authentication for trailing arguments (#1328)
+  * Remove empty file (#1326)
+  * Fix delete user failed on sqlite (#1321)
+  * Fix inconsistency in layout (#1316)
+  * Fix gpg wrong column types (#1303)
+  * Fix wiki bugs (#1294)
+  * Fix missing less sources for oauth (#1288)
+  * Make sure both scripts/ can live side by side (#1264)
+  * Fix nil-dereference bug (#1258)
```
Comment (davydov-vyacheslav) [inline CHANGELOG.md:217]: I don't know how it is going in current project, but from my point of view, I believe that in current list of fixes should be only fixes for defects of previous versions. As end-user I shouldn't know what issues was during features' implementation and how they were fixed

Comment (sapk) [reply CHANGELOG.md:217]: I think that it is auto-generated. We could in the blog post highlight some new feature and big fix and keep this list exhausitive.

**Comment by sapk** [comment]
Trusted LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (lunny) [reply CHANGELOG.md:217]: @davydov-vyacheslav when this posted to github's release, all the #xxxx will be linked automatically to the pull request page.

Comment (davydov-vyacheslav) [reply CHANGELOG.md:217]: ok. thank you for clarification

**Comment by lafriks** [comment]
LGTM

**Pull request merged**